techbuzz/src/main/resources/templates/fragments/post.html

```diff
@@ -0,0 +1,54 @@
+<div th:fragment="post" th:id="'post-container-' + ${post.id}">
```

Comment:
You can add the following HTML declaration in **post.html**, it will help to give autocompletion in IDE.

```
<!DOCTYPE html>
<html lang="en"
      xmlns="http://www.w3.org/1999/xhtml"
      xmlns:th="http://www.thymeleaf.org"
      xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity">
<body>
<div th:fragment="post" th:id="'post-container-' + ${post.id}">
...
...
</div>
</body>
</html>
```